README.md

```diff
@@ -1,3 +1,12 @@
+<p align="center">                                                                                                                                            ─╯
```

Comment:
@JoseMotto-sg, please remove spaces and text character at the end of this line.

Reply:
done